### PR TITLE
In 137112317 extract es to api

### PR DESCRIPTION
**Pull request by noraOSI**

### CODE REVIEW

File: app/repositories/screening_repository.rb [hunk 1]
```diff
@@ -28,6 +29,14 @@ def self.update(screening)
     Screening.new(response.body)
   end
 
+  def self.search(query)
+    response = make_api_call(SCREENINGS_SEARCH_PATH, :get, query)
```
Comment (pemaOSI) [inline app/repositories/screening_repository.rb]: can we not call this query?

File: app/repositories/screening_repository.rb [hunk 2]
```diff
@@ -29,8 +28,8 @@ def self.update(screening)
     Screening.new(response.body)
   end
 
-  def self.search(query)
-    response = make_api_call(SCREENINGS_SEARCH_PATH, :get, query)
+  def self.search(search_terms)
+    response = make_api_call("#{SCREENINGS_PATH}?#{search_terms.to_query}", :get)
```
Comment (pemaOSI) [inline app/repositories/screening_repository.rb]: why to_query?

**Comment by pemaOSI** [comment]
Word query is being used in search, methods named with query, and test reflects query. Can we change that appropriately?

File: app/repositories/screening_repository.rb (hunk 2 above)
Comment (noraOSI) [inline app/repositories/screening_repository.rb]: to change it from hash to a query string format, so the url looks like `/api/v1/screenings_search?response_times%5B%5D=immediate&response_times%5B%5D=within_twenty_four_hours`

File: app/repositories/screening_repository.rb (hunk 1 above)
Comment (noraOSI) [inline app/repositories/screening_repository.rb]: we changed it to search term in the later commit

**Pull request closed without merging**